modern/src/runtime/AnimatedLayer.ts

```diff
@@ -2,6 +2,97 @@ import Layer from "./Layer";
 import { unimplementedWarning } from "../utils";
 
 class AnimatedLayer extends Layer {
+  _playing: boolean;
+  _frameNum: number;
+
+  constructor(node, parent, annotations, store) {
+    super(node, parent, annotations, store);
+
+    if (!node.attributes.hasOwnProperty("autoplay")) {
```

Comment:
Why `.hasOwnProperty()` and not `!= null`?

Reply:
I figured that since we were testing for whether or not the property existed to fill it in with the default value that we should use `hasOwnProperty`? Is there some reason that `!= null` is better in this scenario?

Reply:
I guess I used different versions in different places. gonna make it consistent hah